### ISSUE TITLE
3.1 Make CNN multilabel

### ISSUE THREAD

**Issue by hughmancoder** [hughmancoder/ML-based-analysis-of-sound#3]
Multi-label systems use a Sigmoid activation with Binary Cross-Entropy Loss,
- [ ] Create a new finle called train_multi_label.ipynb, using modules from train_utils where relevant
- [ ] Add sigmoid head + BCEWithLogitsLoss in PyTorch 
- [ ] the target output vector can be a multi-hot vector to indicate the predicted class
